src/Sylius/Bundle/CoreBundle/Form/Extension/ChannelTypeExtension.php

```diff
@@ -66,13 +66,18 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ])
             ->add('taxCalculationStrategy', TaxCalculationStrategyChoiceType::class, [
                 'label' => 'sylius.form.channel.tax_calculation_strategy',
-            ])
-            ->add('themeName', ThemeNameChoiceType::class, [
+            ]);
+
+        if (class_exists('\Sylius\Bundle\ThemeBundle\Form\Type\ThemeNameChoiceType')) {
```

Comment:
we could just add another extension in ShopBundle, which will be automatically executed if it is enabled